src/libs/portfolio/portfolio.ts

```diff
@@ -306,7 +306,7 @@ export class Portfolio {
       tokenErrors: tokensWithErrResult
         .filter(([error, result]: [string, TokenResult]) => error !== '0x' || result.symbol === '')
         .map(([error, result]: [string, TokenResult]) => ({ error, address: result.address })),
-      collections: collections.filter((x) => x.collectibles?.length)
+      collections
```

Comment:
Why?

Reply:
So we get collection info regardless if there is any changes to nft ownership. 
This check is moved to the dashboard component so as we need it only there